octopoes/octopoes/config/settings.py

```diff
@@ -55,40 +55,45 @@ class Settings(BaseSettings):
-    xtdb_uri: AnyHttpUrl = Field(..., example="http://crux:3000", description="XTDB API", env="XTDB_URI")
+    queue_uri: AmqpDsn = Field(..., examples=["amqp://"], description="KAT queue URI", validation_alias="QUEUE_URI")
+    xtdb_uri: AnyHttpUrl = Field(
+        ..., examples=["http://crux:3000"], description="XTDB API", validation_alias="XTDB_URI"
+    )
     xtdb_type: XTDBType = Field(
         XTDBType.XTDB_MULTINODE,
         description="Determines how Octopoes will format documents' primary in serialization (crux.db/id vs xt/id)",
         possible_values=["crux", "xtdb", "xtdb-multinode"],
     )
 
     katalogus_api: AnyHttpUrl = Field(
-        ..., example="http://localhost:8003", description="Katalogus API URL", env="KATALOGUS_API"
+        ..., examples=["http://localhost:8003"], description="Katalogus API URL", validation_alias="KATALOGUS_API"
     )
 
     scan_level_recalculation_interval: int = Field(
         60, description="Interval in seconds of the periodic task that recalculates scan levels"
     )
-    bits_enabled: Set[str] = Field(set(), example='["port-common"]', description="Explicitly enabled bits")
-    bits_disabled: Set[str] = Field(set(), example='["port-classification-ip"]', description="Explicitly disabled bits")
+    bits_enabled: Set[str] = Field(set(), examples=['["port-common"]'], description="Explicitly enabled bits")
+    bits_disabled: Set[str] = Field(
+        set(), examples=['["port-classification-ip"]'], description="Explicitly disabled bits"
+    )
 
     span_export_grpc_endpoint: Optional[AnyHttpUrl] = Field(
-        None, description="OpenTelemetry endpoint", env="SPAN_EXPORT_GRPC_ENDPOINT"
+        None, description="OpenTelemetry endpoint", validation_alias="SPAN_EXPORT_GRPC_ENDPOINT"
     )
 
-    class Config:
-        env_prefix = "OCTOPOES_"
-
-        @classmethod
-        def customise_sources(
-            cls,
-            init_settings: SettingsSourceCallable,
-            env_settings: SettingsSourceCallable,
-            file_secret_settings: SettingsSourceCallable,
-        ) -> Tuple[SettingsSourceCallable, ...]:
-            backwards_compatible_settings = BackwardsCompatibleEnvSettings()
-            return env_settings, init_settings, file_secret_settings, backwards_compatible_settings
+    model_config = SettingsConfigDict(env_prefix="OCTOPOES_")
+
+    @classmethod
+    def settings_customise_sources(
+        cls,
+        settings_cls: Type[BaseSettings],
+        init_settings: PydanticBaseSettingsSource,
+        env_settings: PydanticBaseSettingsSource,
+        dotenv_settings: PydanticBaseSettingsSource,
+        file_secret_settings: PydanticBaseSettingsSource,
+    ) -> Tuple[PydanticBaseSettingsSource, ...]:
+        backwards_compatible_settings = BackwardsCompatibleEnvSettings(settings_cls)
+        return env_settings, init_settings, file_secret_settings, backwards_compatible_settings
```

Comment:
Dotenv_settings is missing and the order is changed.

```suggestion
        return init_settings, env_settings,  dot_env_settings, file_secret_settings, backwards_compatible_settings
```